src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ItemsRepeater.cs

```diff
@@ -608,9 +608,11 @@ internal int Indent()
 
 		void OnLoaded(object sender, RoutedEventArgs args)
 		{
+#if !HAS_UNO // With uno we always detach from the scroller on unload so we need to force a layouting pass to re-subscribe to scroller and update viewport (in a single pass)
```

Comment:
Is this always false?

https://github.com/unoplatform/uno/blob/6f1d732fd0fa79f6994b01694e9f86e75c7ccd39/src/Uno.CrossTargetting.targets#L98-L100

I guess Uno.UI isn't built against `uap10.0.18362` or `net5.0-windows10.0.18362.0` so we always have `HAS_UNO` defined? I could definitely be missing something

Reply:
perhaps, we are preserving original winui source, but "commenting them out" through the #if?

Reply:
In that case I think `#if false` would be clearer

Reply:
Yes it's always `false`. The idea is just to disable that code but keep it for reference as it's coming from WinUI, so it will be easier to update by comparing WinUI source code. It also makes clearer what the specific changes we made when we port code from WinUI.